### DISCUSSION TITLE
stuttering behavior when direct streaming

### DISCUSSION THREAD

**Discussion by scooterama** [ChristopherKlay/StadiaEnhanced#250] [General]
sometimes (in certain games it's reproducible) there's a Problem when streaming a Game to YouTube and it becomes unplayably laggy. 

tried with Little Nightmares II twice and Assassins Creed Origins.
when turning off SE it's perfect (but not in 4K/1440p on YouTube) anymore.

**Comment by ChristopherKlay** [comment]
I've looked into this, but can't reproduce it.

Currently checking if someone with a 4K screen and capable GPU/iGPU can confirm that this is a extension issue and not an issue with those games being livestreamed via the 4K stream specifically.
